modules/reindex/src/main/java/org/elasticsearch/index/reindex/TransportStartReindexTaskAction.java

```diff
@@ -87,7 +103,7 @@ protected void doExecute(Task task, StartReindexTaskAction.Request request, Acti
 
         // In the current implementation, we only need to store task results if we do not wait for completion
         boolean storeTaskResult = request.getWaitForCompletion() == false;
-        ReindexTaskParams job = new ReindexTaskParams(storeTaskResult, included);
+        ReindexTaskParams job = new ReindexTaskParams(storeTaskResult, resolveIndexPatterns(request.getReindexRequest()), included);
```

Comment:
Do we want to be putting this in the cluster state? I don't have a sense for how large this gets, but I assume it could go in the index?

Reply:
Thanks, this is clearly wrong, fixed in 33d8fd4ce0f41b8bf43453feacfc74108d618fe2.